src/applications/disability-benefits/2346/containers/ConfirmationPage.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import moment from 'moment';
+import { connect } from 'react-redux';
+import Scroll from 'react-scroll';
+
+import { focusElement } from '../../../../platform/utilities/ui';
+
+const scroller = Scroll.scroller;
+const scrollToTop = () => {
+  scroller.scrollTo('topScrollElement', {
+    duration: 500,
+    delay: 0,
+    smooth: true,
+  });
+};
+
+export class ConfirmationPage extends React.Component {
+  componentDidMount() {
+    focusElement('.schemaform-title > h1');
+    scrollToTop();
+  }
+
+  render() {
+    const { submission, data } = this.props.form;
+    const { response } = submission;
+    const name = data.veteranFullName;
```

Comment:
When doing dot properties it's best to code this defensively and use something like:
```jsx
    const { form } = this.props;
    const response = form?.submission?.response;
    const veteranFullName = form?.data?.veteranFullName;
```